targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/TOOLCHAIN_ARMC6/musca_ns.sct

```diff
@@ -0,0 +1,64 @@
+#! armclang --target=arm-arm-none-eabi -march=armv8-m.main -E -xc
+
+/*
+ * Copyright (c) 2019-2020 Arm Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../partition/region_defs.h"
+#include "../cmsis_nvic.h"
+
+#if !defined(MBED_ROM_START)
+  #define MBED_ROM_START    NS_CODE_START               // 0x0A080400
+#endif
+
+#if !defined(MBED_ROM_SIZE)
+  #define MBED_ROM_SIZE     NS_CODE_SIZE                // 0x7f400
+#endif
+
+#if !defined(MBED_RAM_START)
+  #define MBED_RAM_START    NS_DATA_START               // 0x20040000
+#endif
+
+#if !defined(MBED_RAM_SIZE)
+  #define MBED_RAM_SIZE     NS_DATA_SIZE                // 0x40000
+#endif
+
+#define VECTOR_SIZE         NVIC_RAM_VECTOR_SIZE
+
+#if !defined(MBED_BOOT_STACK_SIZE)
+  #define MBED_BOOT_STACK_SIZE      0x400
```

Comment:
After this was copy-pasted from B1, we've changed the B1 sct file in how it allocates stack. To ensure proper functioning with ARMC, you probably want to make a follow up PR with the same changes.

Reply:
Thanks for the heads-up!
Could you please point me to the B1 change?

Reply:
Apologies, as I thought the PR was merged already. I'll update B1 and S1 in my PR as your PR is likely to merge first. https://github.com/ARMmbed/mbed-os/pull/13452

Cheers

Reply:
Looks my PR got merged first. The B1 linker script changes are contained in the following commits (on `master`):
- https://github.com/ARMmbed/mbed-os/commit/612b148fd4d49b60a7c9c605fef051d147978f7a
- https://github.com/ARMmbed/mbed-os/commit/39e69d328d136678665bae36b6245ae839bcd403

Reply:
I have implemented the changes on the S1 files.